examples/Cargo.toml

```diff
@@ -9,6 +9,7 @@ edition = "2018"
 default = []
 metal = ["gfx-backend-metal"]
 gl = ["gfx-backend-gl"]
```

Comment:
Examples don't need to try to cut out the dependencies. Example project can just always enable x11 like it does for vulkan, no need to show this out in the interface.

Reply:
That makes sense. Done!